### PR TITLE
Feature/33 projects list

### PR DESCRIPTION
**Pull request by ela-ocana-jalafund**
[AB#33](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/33)

### CODE REVIEW

File: src/components/Project/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Description, Email, CalendarToday } from "@material-ui/icons";
+import "./Card.scss";
+
+function Card(props) {
+  const { id, name, username, email, address } = props;
```
Comment (DaleneyRuben) [inline src/components/Project/Card.jsx]: can you please order this alphabetically?

File: src/components/Project/Projects.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  const classes = useStyles();
+  const [projects, setProjects] = useState([]);
+
+  const refresh = () => {
+    let list = [];
+    GetProjects()
+      .then((result) => {
+        list = result;
+      })
+      .then(() => setProjects(list));
```
Comment (DaleneyRuben) [inline src/components/Project/Projects.jsx]: can you try changing this to use async/await approach? It's better to use async/await over promises

File: src/components/Project/Projects.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  };
+  useEffect(refresh, []);
```
Comment (DaleneyRuben) [inline src/components/Project/Projects.jsx]: NABD but you're not reusing the refresh function so you can move all that code here

File: src/components/Project/ProjectsAPI.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
+
+export default async function GetProjects() {
```
Comment (DaleneyRuben) [inline src/components/Project/ProjectsAPI.js]: Only functional components starts with uppercase, **getProjects** is the correct name.

File: src/components/Project/ProjectsAPI.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  let errorHandler = true;
+  try {
+    let result = await (await instance.get("/users")).data;
```
Comment (DaleneyRuben) [inline src/components/Project/ProjectsAPI.js]: Why the double await? I think the following would work:
```
const result = await instance.get("/users").data
```

**Review by DaleneyRuben — CHANGES_REQUESTED** [review]

File: src/components/Project/ProjectsAPI.js (hunk 5 above)
Comment (ela-ocana-jalafund) [reply src/components/Project/ProjectsAPI.js]: @DaleneyRuben I tried it but it doesn't work. Double async await is similar to:
const response = await axios.get("https://jsonplaceholder.typicode.com/users");
const result = await response.data;

File: src/components/Project/ProjectsAPI.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+export default async function getProjects() {
+  let handlerError = false;
+  try {
+    const response = await axios.get(
+      "https://jsonplaceholder.typicode.com/users"
+    );
+    const result = await response.data;
```
Comment (DaleneyRuben) [inline src/components/Project/ProjectsAPI.js]: I don't think you need the await here
```
const { data: result } = await axios.get(
      "https://jsonplaceholder.typicode.com/users"
    );
```
You created an axios instance before I think that's why you had to use the double await

**Comment by DaleneyRuben** [comment]
Please @ela-ocana-jalafund  update your branch with develop. so in the App component you can integrate your Projects component
```
  ...
<Route component={Home} exact path="/"></Route>
<Route component={Projects} exact path="/projects">
  ...
```

**Review by gustavo-rojas-jalafund — DISMISSED** [review]

**Review by jorge-lema-jalafund — DISMISSED** [review]

**Review by alejandro-castellon — DISMISSED** [review]

**Review by DaleneyRuben — APPROVED** [review]
All looking good 🚀

**Review by alejandro-castellon — APPROVED** [review]

**Review by gustavo-rojas-jalafund — APPROVED** [review]

**Pull request merged**